vnext/Microsoft.ReactNative.ProjectReunion/Microsoft.ReactNative.ProjectReunion.csproj

```diff
@@ -0,0 +1,42 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net5.0-windows10.0.19041.0</TargetFramework>
+    <Platforms>x64</Platforms>
```

Comment:
I am not sure on the latest story in Reunion for Platforms is, but for the projection DLL, the below is what the MRT projection does.  You should consider either specifying Platforms as AnyCPU or specifying PlatformTarget as AnyCPU similar to what MRT does.  Otherwise, consumers will get errors indicating it was built for the wrong architecture when using something other than x64.

https://github.com/microsoft/ProjectReunion/blob/c0430c8910978c2eba09db38c807195740c6049b/dev/MRTCore/mrt/Microsoft.Windows.ApplicationModel.Resources/projection/Microsoft.Windows.ApplicationModel.Resources.Projection.csproj#L6-L7

Reply:
The thing is the code for Microsoft.ReactNative.dll (that this project projects onto C#) is arch specific. The idea was for now to start with x64 but if there is a way for the projection to be built for anycpu, great. AFAIK, anycpu is not working/supported in reunion? unclear on the details but see microsoft/ProjectReunion#922